media-sound/yoshimi/yoshimi-1.7.1.ebuild

```diff
@@ -27,7 +27,7 @@ DEPEND="
 	virtual/jack
 	x11-libs/cairo[X]
 	x11-libs/fltk:1[opengl]
-	lv2? ( media-libs/lv2 )
+	lv2? ( <media-libs/lv2-1.18.0 )
```

Comment:
Looks good, but this needs a revbump. Ideally it'd be in a separate commit, too. You can use `git mv` here.

Reply:
Thanks, I don't know that media-libs/lv2 is updated.